Deployment Tracking Doc Update (#14929)
DataDog/documentation: content/en/tracing/services/deployment_tracking.md

### Metric definition

`datadog.service.time_between_deployments{env, service, second_primary_tag}`
: **Prerequisite:** This metric exists for any APM service with version tagging enabled through [Unified Service Tagging][1].<br>
**Description:** The time in seconds elapsed between a deployment of a service and the deployment of the most recent version prior to that.<br>
**Metric type:** [Distribution][2]<br>
**Tags:** The metric is tagged with the service's `env`, `service`, and [second primary tag][3].

### Examples

If you have a service that deploys version A at time = 0 and version B at time = 10, then the value of the metric `datadog.service.time_between_deployments` is 10:

Time = 0
: `{service: foo, env: prod, cluster-name: dev-shopist, version: A}`

Time = 10
: `{service: foo, env: prod, cluster_name: dev-shopist, version: B}`

Time between deployments
: `datadog.service.time_between_deployments{env: prod, cluster_name: dev-shopist} = 10`

> If you deploy version X at time = 20 on cluster `dev-shopist`, version Y at time = 30 on cluster `us-staging`, and version Y again at time = 45 on cluster `dev-shopist`, the `max` value of the metric `datadog.service.time_between_deployments` for an ambiguous cluster is 25 (the time of the most recent Y minus the last X): 

Reviewer: ```suggestion
If you deploy version X at time = 20 on cluster `dev-shopist`, version Y at time = 30 on cluster `us-staging`, and version Y again at time = 45 on cluster `dev-shopist`, the `max` value of the metric `datadog.service.time_between_deployments` for any cluster is 25 (the time of the most recent Y minus the last X): 
```
